backend/btrixcloud/operator.py

```diff
@@ -1362,9 +1401,13 @@ async def update_crawl_state(self, redis, crawl, status, pods, done) -> CrawlSta
                 )
                 return status
 
-            completed = status.pagesDone and status.pagesDone >= status.pagesFound
+            # completed = status.pagesDone and status.pagesDone >= status.pagesFound
 
-            state = "complete" if completed else "partial_complete"
+            # state = "complete" if completed else "partial_complete"
+            if status.stopReason in ("stopped_by_user", "stopped_quota_reached"):
+                state = status.stopReason
```

Comment:
Do we want `size-limit` and `time-limit` to be set as `state`, or should they result in a `complete` status? As I'm reading this, they could currently get set as state?